docs/dependency-injection.md

```diff
@@ -240,9 +240,9 @@ Produces responses like the following at "/":
 
         A2: 139976289979936
 
-        B1: 139976289979984
+        B1: 139976289979988
```

Comment:
This change is to make it easier to tell that these B1 and B2 numbers are different from the B1 and B2 numbers in the first request. Because when I initially tried to compare them, I only compared a few of the initial and end digits, and the numbers looked the same, and I only realized that they were actually different after reviewing them a second time.